x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/services/analytics_service/get_analytics.test.ts

```diff
@@ -15,6 +15,7 @@ describe('get_analytics', () => {
     const mockResponse: GetDataFrameAnalyticsStatsResponseOk = {
       count: 2,
       data_frame_analytics: [
+        // @ts-expect-error @elastic-elasticsearch Data frame types incomplete
```

Comment:
this comment is confusing.
It's not that the types are wrong, the error is caused by this test response missing lots of properties.

Reply:
Good catch - updated in f62dad344f36fd8fcd405c98394d91606b37be08